frontend/packages/dev-console/src/components/monitoring/overview/MonitoringTab.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { DeploymentKind } from '@console/internal/module/k8s';
+import { OverviewItem } from '@console/shared';
+import MonitoringMetricsSection from './MonitoringMetricsSection';
+
+const MonitoringTab: React.FC<MonitoringTabProps> = ({ item }) => {
+  const { obj: d } = item;
+  return (
+    <div className="overview__sidebar-pane-body">
```

Comment:
for class name we follow convention `odc-filename-*

Reply:
Agree. This class exists in overview component in /public & gets loaded before Monitoring Tab.